core/src/main/java/jenkins/util/xstream/XStreamDOM.java

```diff
@@ -540,6 +543,7 @@ public XStreamDOM unmarshalElement(HierarchicalStreamReader r, UnmarshallingCont
         }
     }
 
-    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
-    public static XmlFriendlyReplacer REPLACER = new XmlFriendlyReplacer();
+    @Restricted(NoExternalUse.class)
+    @RestrictedSince("2.TODO")
+    public static final XmlFriendlyReplacer REPLACER = new XmlFriendlyReplacer();
```

Comment:
I guess I forgot to search for this one with `usage-in-plugins`. Can you verify this is unused with `usage-in-plugins`? If that is asking too much, I can look again when I have some time.